Source/WebKit/WebProcess/WebPage/gtk/AcceleratedSurfaceDMABuf.cpp

```diff
@@ -105,15 +97,16 @@ uint64_t AcceleratedSurfaceDMABuf::surfaceID() const
 void AcceleratedSurfaceDMABuf::clientResize(const WebCore::IntSize& size)
 {
     auto& display = WebCore::PlatformDisplay::sharedDisplayForCompositing();
-    if (m_backImage) {
-        display.destroyEGLImage(m_backImage);
-        m_backImage = nullptr;
-    }
 
-    if (m_frontImage) {
-        display.destroyEGLImage(m_frontImage);
-        m_frontImage = nullptr;
-    }
+    if (m_back.image)
+        display.destroyEGLImage(m_back.image);
+    if (m_back.colorBuffer)
+        glDeleteRenderbuffers(1, &m_back.colorBuffer);
+
+    if (m_front.image)
+        display.destroyEGLImage(m_front.image);
+    if (m_front.colorBuffer)
+        glDeleteRenderbuffers(1, &m_front.colorBuffer);
```

Comment:
I think we should clear m_back and m_front since we can early return below in cases of errors